tapdb/sqlc/migrations/000013_assets_unique_constraint.up.sql

```diff
@@ -0,0 +1,31 @@
+-- We will apply a new unique constraint on the assets table. This constraint
```

Comment:
nit: we should add a comment to the `.down.sql` file that it's empty on purpose.